.meta/links.toml

```diff
@@ -155,8 +155,9 @@ logplex = "https://devcenter.heroku.com/articles/logplex"
 logplex_protocol = "https://github.com/heroku/logplex/blob/master/doc/README.http_drains.md"
 lua = "https://www.lua.org/"
 lua_docs = "https://www.lua.org/manual/5.3/"
-lua_manual = "http://www.lua.org/manual/5.1/manual.html"
-lua_require = "http://www.lua.org/manual/5.1/manual.html#pdf-require"
+lua_manual = "https://www.lua.org/manual/5.1/manual.html"
```

Comment:
Does it make sense to ensure all of these are referring to the same Lua version?

Reply:
I'll clean this up, the version we use is 5.3 and the linked manual has to correspond to it. I wonder does it make sense to also link to https://www.lua.org/pil/, which is great, although the edition [freely available online](https://www.lua.org/manual/5.1/) is only for Lua 5.1.

Reply:
You could add this to a "Learning Lua" section under "How It Works". I think we'll also want a Lua guide in our new `/guides` section, which might be a better place for referencing that.